src/pyastar2d/astar_wrapper.py

```diff
@@ -19,14 +19,30 @@
     ctypes.c_int,   # start index in flattened grid
     ctypes.c_int,   # goal index in flattened grid
     ctypes.c_bool,  # allow diagonal
+    ctypes.c_int,   # heuristic_override
+    ctypes.c_float, # tiebreaker_coefficient
 ]
 
 
 def astar_path(
         weights: np.ndarray,
         start: Tuple[int, int],
         goal: Tuple[int, int],
-        allow_diagonal: bool = False) -> Optional[np.ndarray]:
+        allow_diagonal: bool = False,
+        heuristic_override: int = 0,
+        tiebreaker_coefficient: float = 0.0) -> Optional[np.ndarray]:
+    """
+    Run astar algorithm on 2d weights.
+
+    param np.ndarray weights: A grid of weights e.g. np.ones((10, 10), dtype=np.float32)
+    param Tuple[int, int] start: (x, y) point to start
+    param Tuple[int, int] goal: (x, y) point to end
+    param bool allow_diagonal: Whether to allow diagonal moves
+    param int heuristic_override: Override heuristic 0=auto, 1=diagonal, 2=manhattan, 3=orthogonal-x, 4=orthogonal-y
+    param float tiebreaker_coefficient: Add tiebreaker to heuristic cost, 0=disable, positive enables it by that amount
+
+    Important: Please take care when using heuristic_override and tiebreaker_coefficient, they may not take the shortest path.
```

Comment:
As above I don't really like the tiebreaker coefficient:
1. I think we can get rid of the argument by using the heuristic function's value to break ties directly (see another comment)
2. I don't like breaking the admissibility of the heuristic and thus the shortest-path guarantee


If you really need it, I might suggest moving it to an `astar_path_experimental` function or similar to make it clear to the user that they're using this at their own risk, but I would prefer getting rid of it.